src/coreclr/jit/instr.cpp

```diff
@@ -945,7 +945,7 @@ void CodeGen::inst_RV_RV_IV(instruction ins, emitAttr size, regNumber reg1, regN
 {
     assert(ins == INS_shld || ins == INS_shrd || ins == INS_shufps || ins == INS_shufpd || ins == INS_pshufd ||
            ins == INS_cmpps || ins == INS_cmppd || ins == INS_dppd || ins == INS_dpps || ins == INS_insertps ||
-           ins == INS_roundps || ins == INS_roundss || ins == INS_roundpd || ins == INS_roundsd);
+           ins == INS_roundps || ins == INS_roundss || ins == INS_roundpd || ins == INS_roundsd || ins == INS_pextrd);
```

Comment:
Looks like the HWIntrinsics go through `inst_RV_TT_IV` which is why this wasn't asserted here: https://github.com/dotnet/runtime/blob/16c48d6a45d4d706c327fc840e1a5d450fb189ae/src/coreclr/jit/hwintrinsiccodegenxarch.cpp#L1550